Contributing.md

```diff
@@ -0,0 +1,88 @@
+# Contributing
+
+🎊 🎉 You're Awesome!! 🎉 🎊
+
+Thank you for considering a contribution to our project.
+
+## Installation
+
+If you have never set up Jekyll or Ruby in your local environment, we recommend the Jekyll docs for [installing on your platform.](https://jekyllrb.com/docs/installation/)
+
+We use the [latest stable Ruby version for development](https://github.com/hurricane-response/response-theme/blob/master/.ruby-version)
+
+We also use [Gulp](https://gulpjs.com) for our build tool. Though Jekyll has support for Sass, we have opted to write plain CSS for 2019 browsers and let the tools backfill for compatibility. If you haven't worked with Gulp before, we recommend their [getting started docs](https://gulpjs.com/docs/en/getting-started/quick-start) which includes a link to [Node install steps](https://nodejs.org/en/)
+
+
+Clone the repository
+
+```{bash}
+git clone git@github.com:hurricane-response/response-theme.git
+```
+
+navigate to the directory
+
+```{bash}
+cd response-theme
+```
+
+install the Jekyll dependencies
+
+```{bash}
+bundle install
+```
+
+install the Gulp dependencies
+
+```{bash}
+npm i
+```
+
+`Gulpfile` has two main tasks: `build` and `watch`
+
+`build` does just that - builds the `_site` directory.
+
+`watch` watches both `assets/css/_inc` directory to check for CSS changes as well as Jekyll file changes and rebuilds the appopriate files.
+
+**A note about the CSS** All CSS is in the `_inc` directory as plain CSS. Current browser support for variables is strong, but we use PostCSS to fill in browser compatibility. PostCSS also compiles `assets/css/_inc/main.css` into `assets/css/main.css`. You should never need to edit `assets/css/main.css` file directly.
+
+Reminder that this is a [Jekyll theme](https://jekyllrb.com/docs/themes/), not a stand alone site. We optimized the build tools to both allow customization by the individual response sites by overriding `main.css` locally as well as is.
+
+## Where to start?
+
+The [issues](https://github.com/hurricane-response/response-theme/issues). We ask that you create an issue before starting work on something, even if a new feature. We also ask that you adhere as best as possible to the issue template to help us better triage. Certainly small,minor typos do not require a novel, but the more information the better.
+
+We have found that robust discussion before starting a new feature or large refactor has the greatest chance of success. Discussions can occur in the issue created or in Slack.
+
+
+**Note** sometimes during an active response focus is on immediate needs and those of our partners. We respectfully ask that you use the issues and `@` one of the contributors.
+
+### Whoa! I just want to fix a typo.
```

Comment:
This seems a bit disconnected with the prior section -- it doesn't address the need to create an issue prior to fixing the typo via a PR through the web interface. Are we giving a pass on creating an issue for typo fixes, or do we still want folks to submit one? If we do, I think we should indicate that here. If not, we should explicitly say so.

Reply:
I wasn't sure where to fit that in, but I wanted to make sure for those folks who want to fix a typo or small change weren't discouraged by all of the talk of Ruby/Node and pull requests. I should add to please still open an issue, and the next section is Commit Message leading into Pull Requests.